Source/WebCore/loader/ResourceLoadInfo.h

```diff
@@ -55,9 +55,11 @@ enum class ResourceType : uint32_t {
     Fetch = 0x0200,
     WebSocket = 0x0400,
     Other = 0x0800,
-    CSPReport = 0x10000,
+    CSPReport = 0x100000,
+    TopDocument = 0x200000,
+    ChildDocument = 0x400000,
```

Comment:
Since we are changing the version number, we can group these bits back together. I sent you a version of this file, since there is more to change than just these values.
```suggestion
    CSPReport = 0x1000,
    TopDocument = 0x2000,
    ChildDocument = 0x4000,
```